.github/workflows/multiqc.yml

```diff
@@ -0,0 +1,30 @@
+name: multiqc
+on:
+  push:
+    paths:
+      - software/multiqc/**
+      - .github/workflows/multiqc.yml
+      - tests
+  pull_request:
+    paths:
+      - software/multiqc/**
+      - .github/workflows/multiqc.yml
+      - tests
+
+jobs:
+  ci_test:
+    runs-on: ubuntu-latest
+    env:
+      NXF_ANSI_LOG: false
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Install Nextflow
+      run: |
+        export NXF_VER="20.07.1"
+        wget -qO- get.nextflow.io | bash
+        sudo mv nextflow /usr/local/bin/
+
+    # Test the module
+    - run: nextflow run ./software/multiqc/test/ -profile docker
```

Comment:
This needs to use pytest-workflow as @drpatelh said. Let me know if you have any questions on it!

Reply:
@Emiller88 , I have rebased on the master and updated the structure accordingly.

Reply:
This still isn't correct, it needs to install pytest-workflow and run pytest-workflow with the multiqc tag.

Reply:
Oh, yeah. I took a deeper look at the `pytest-workflow-refactor` and your recent commits, then I could make more sense of what to do.